spynnaker/pyNN/models/abstract_models/abstract_population_data_spec.py

```diff
@@ -157,12 +156,20 @@ def write_neuron_parameters(
             region=constants.POPULATION_BASED_REGIONS.NEURON_PARAMS.value)
 
         # Write header info to the memory region:
+
+        #if key is None, write boolean saying key is not to be used otherwise
+        # write bool saying key is to be used
+        #if key is None:
+         #   spec.write_value(data=0)
+        #else:
+         #   spec.write_value(data=1)
+
         # Write Key info for this core:
-        population_identity = \
-            packet_conversions.get_key_from_coords(processor_chip_x,
-                                                   processor_chip_y,
-                                                   processor_id)
-        spec.write_value(data=population_identity)
+        #if key is None:
+         #   spec.write_value(data=0)
+        #else:
+        #    spec.write_value(data=key)
```

Comment:
This should be removed if it is no longer useful (remember that git lets you get it back if you really need it)